### ISSUE TITLE
Bug: offshore 'all commodities' bar chart total is not correct

### ISSUE THREAD

**Issue by meiqimichelle** [18F/doi-extractives-data#1973]
On the offshore pages, the total under the bar chart is actually royalties instead of the all-revenue-streams total. ie, in this image, it should be the $4... number, not the $3... number. 

![pasted image at 2016_09_16 10_09 am](https://cloud.githubusercontent.com/assets/4827522/18591876/5221f904-7be9-11e6-9ba5-0ad562ea5a32.png)

**Comment by meiqimichelle** [comment]
Looking further -- the All Commodities bar chart total on all our offshore areas aren't correct, thought they seem to be incorrect in different ways (ie, the others don't seem to be pulling royalties, but not sure what they're doing).

**Comment by gemfarmer** [comment]
Another issue: The Civil Penalties figures didn't make it into the national All Commodities bar chart. For more detail, see https://github.com/18F/doi-extractives-data/pull/1977#issuecomment-247721503

**Comment by gemfarmer** [comment]
Notes on this one. 

The issue here is that we have two different values for the national revenue rollups: `$7,507,964,282` and `$7,499,581,883`.

The difference between the two is `$8,382,399`, which is exactly the sum of Civil Penalties (`$8,035,048`)  and Other Revenues (`$347,351`), both of which were recently added. 

When we look at the sqlite queries, it paints a helpful picture. See that None|All line?? 😍 
```
sqlite> select * from regional_revenue_type where commodity = 'None' and year = '2015';
2015|None|All|8382398.85
2015|None|All|52239171.25
2015|None|Civil Penalties|8035048.31
2015|None|Inspection Fees|52239171.25
2015|None|Other Revenues|347350.54
```

Unfortunately, that sweet, sweet number didn't make it into the `regional_revenue` table _or_ the `national_revenue` table:

```
sqlite> select year, sum(revenue) as revenue from national_revenue where commodity != 'All' and year == '2015';
2015|7499581882.93
sqlite> select year, sum(revenue) as revenue from regional_revenue where commodity != 'All' and year == '2015';
sqlite> select * from regional_revenue where commodity = 'None' and year = '2015';
2015|None|None|offshore|52239171.25
```

So there's the mission: Get Civil Penalties and Other Revenues into the `regional_revenue` table (which creates the `national_revenue` table)!

**Comment by meiqimichelle** [comment]
Yaaaay!

**Comment by meiqimichelle** [comment]
Closed with #1998
